src/main/java/seedu/address/logic/commands/DoctorRemarkCommand.java

```diff
@@ -26,9 +26,9 @@ public class DoctorRemarkCommand extends Command {
             + "by the index number used in the last doctor listing. "
             + "Existing remark will be overwritten by the input.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + PREFIX_REMARK + " [REMARK]\n"
+            + PREFIX_REMARK + "[REMARK]\n"
```

Comment:
for this the square brackets should be around PREFIX_REMARK also right since its optional